Drivers/DataGenerators/DriversFactory.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Person;
+
+namespace Drivers;
+
+public class DriversFactory
+{
+    public static List<Driver> GenerateDrivers(int driversCount)
+    {
+        var drivers = new List<Driver>();
+        for (int i = 0; i < driversCount; i++)
+        {
+            drivers.Add(DataGenerator.CreateDriver());
+        }
```

Comment:
Just to avoid for loop. Make CreateDrivers(int count) method with .Generate(count).